specification/azurelargeinstance/resource-manager/Microsoft.AzureLargeInstance/preview/2023-07-20-preview/azurelargeinstance.json

```diff
@@ -0,0 +1,1106 @@
+{
```

Comment:
Since PUT and Delete are not being supported for these tracked resource types, ARM won't be able to store these resources data in its Cache (or keep itself up-to-date).

I think, you explained during modeling office hours, but if the creation request is not going through ARM via PUT, do these scenarios like reading resources through Azure Portal blade or using Azure Resource Graph (ARG), do these work today?

Reply:
Yes, the reading resources is currently supported due to our hydration process. This process involves a service that checks every 5 minutes to see if the expected resource exists. If it doesn't, the resource is created using an ARM client. 

As discussed in the meeting, we intend to modify this approach in the future. Specifically, we're planning to implement PUT and DELETE functionalities for internal purposes (Customers will not have access to trigger these). However, this change is still in the planning phase.